cmd/minikube/cmd/config/enable.go

```diff
@@ -40,6 +40,9 @@ var addonsEnableCmd = &cobra.Command{
 		if len(args) != 1 {
 			exit.Message(reason.Usage, "usage: minikube addons enable ADDON_NAME")
 		}
+		if ClusterFlagValue() == constants.NoKubernetesVersion {
```

Comment:
`ClusterFlagValue()` returns the profile, so it doesn't make any sense to compare that against a Kubernetes version.